data/projects/zk-email.ts

```diff
@@ -0,0 +1,26 @@
+import { ProjectInterface } from "@/lib/types"
+
+const description = `
+ZK Email is a library that allows for anonymous verification of email signatures while masking specific data. It enables verification of emails to/from specific domains or subsets of domains, as well as verification based on specific text in the email body. This technology can be used for web2 interoperability, decentralized anonymous KYC, or to create interesting on-chain anonymity sets.
```

Comment:
@Metachaser24 What do you think of below for the description.

ZK Email is a library for anonymous verification of emails while selectively revealing specific data. It allows a prover to prove they have received an email from someone, a domain, or contain a specific text in the subject or the body.
This can be used for Web 2.0 interoperability, email recovery solutions of smart accounts, anonymous KYC, or to create interesting applications based on anonymity sets.

Reply:
I think its good i think we can reword one part so take a look at this:

ZK Email is a library for anonymous verification of emails while selectively revealing specific data.  It allows a prover to demonstrate receipt of an email from a specific sender or domain or the presence of certain text in the subject or body. This can be used for Web 2.0 interoperability, email recovery solutions of smart accounts, anonymous KYC, or to create interesting applications based on anonymity sets.

Reply:
@kalidiagne does it

Reply:
@saleel @Metachaser24 copy updated, if everything is fine for you i need one of you to approve this PR so we can merge it